python/testMP.py

```diff
@@ -9,19 +10,21 @@
     parser.add_argument("--port", default=5005, help="OSC server port")
     args = parser.parse_args()
 
-    r = rosary.Rosary(args.ip, args.port)
+    d = dispatcher.Dispatcher()
+    r = rosary.Rosary(args.ip, args.port, d)
```

Comment:
If you remember, this is the cause of that error that prompted me to say that my stuff wasn't playing nicely with your stuff, aka
`* ROSARY PATHS *
/rosary/pause
Traceback (most recent call last):
  File "testMP.py", line 12, in <module>
    r = rosary.Rosary(args.ip, args.port)
  File "/home/yunfan/jdb/MegaPrayer-oscled-alt/python/mp/rosary.py", line 116, in __init__
    self.register_with_dispatcher()
  File "/home/yunfan/jdb/MegaPrayer-oscled-alt/python/mp/rosary.py", line 408, in register_with_dispatcher
    self.dispatcher.map(osc_path,
AttributeError: 'NoneType' object has no attribute 'map'`

I'm still considering maybe just initializing a dispatcher.Dispatcher() object in the rosary if it's not passed...BUT I don't think this should affect you moving forward? So I might just leave it.

In other words, if this is the last time in your life you have to initialize a Rosary object I might just take the lazy route.